src/modules/CVs/components/cv-details-form/cv-details-form.tsx

```diff
@@ -0,0 +1,116 @@
+import styles from "./cv-details-form.module.css";
+
+const CvDetailsForm = () => {
+    const currentUser = useAuthUser();
+    if (!currentUser) {
+        return <Navigate to={routes.auth.root} replace />;
+    }
+    const { showNotification } = useNotificationContext();
+    const { t } = useTranslation();
+    const [isDisabled, setIsDisabled] = useState(false);
+
+    const cvDetailsSchema = () => {
+        return z.object({
+            name: z.string(),
+            education: z.string(),
+            description: z.string().min(6, {
+                message: t(`Description should be at least ${6} symbols long`),
+            }),
+        });
+    };
+
+    type cvDetailsSchemaType = z.infer<ReturnType<typeof cvDetailsSchema>>;
+
+    const { cvId } = useParams<RouteParams>();
+    if (!cvId) {
+        throw new Error("No cv id");
+    }
+
+    const { data: cvData, loading: loadingCv } = useCvById(cvId);
+    const [updateCv, { loading: updatingCv }] = useCvUpdate();
+
+    useEffect(() => {
+        if (cvData) {
+            setIsDisabled(currentUser.email !== cvData.cv.user.email);
+        }
+    }, [currentUser, cvData]);
+
+    const handleSubmit = async (values: cvDetailsSchemaType) => {
+        try {
+            return await updateCv({
+                variables: {
+                    cv: {
+                        cvId,
+                        name: values.name,
+                        education: values.education,
+                        description: values.description,
+                    },
+                },
+            });
+        } catch (error) {
+            if (error instanceof Error) {
+                showNotification("error", error.message);
+            } else {
+                showNotification("error", "Error updating CV");
+            }
+        }
+    };
+
+    if (loadingCv) return <p>Loading...</p>;
+
+    return (
+        <Flex className={styles.cv_details_form_wrapper}>
+            <Form
+                className={styles.cv_details_form}
+                schema={cvDetailsSchema()}
+                onSubmit={handleSubmit}
+                defaultValues={{
+                    name: cvData?.cv.name ?? "",
+                    education: cvData?.cv.education ?? "",
+                    description: cvData?.cv.description ?? "",
+                }}
+            >
+                <FormTextField
+                    name={"name"}
+                    placeholder={t("Name")}
+                    size="large"
+                    readOnly={isDisabled}
+                />
+                <FormTextField
+                    name={"education"}
+                    placeholder={t("Education")}
+                    size="large"
+                    readOnly={isDisabled}
+                />
+                <FormTextarea
+                    className={styles.cv_details_form_textarea}
+                    name={"description"}
+                    placeholder={t("Description")}
```

Comment:
Also is using just placeholders good for user's experience? Maybe should add atleast some labels? (even if the current (antd-based) input labels don't really look/behave like the MUI ones)

Reply:
I added labels and changed the form layout so that the labels are placed above the inputs instead of to the left of them